doc/version_history.rst

```diff
@@ -6,6 +6,22 @@
 Version History
 ###############
 
+v0.24.0
+-------
+
+* Remove support for the sync_pattern field in low-level commands.
+  This requires ts_hexrotcomm 0.28.
+* Remove support for older (pre-ts_xml 9.2) actuators telemetry.
+
+Requires:
+
+* ts_hexapod_controller 1.2.4
```

Comment:
The version of `ts_hexapod_controller` now is 1.2.5. Please consider to use a higher version number such as 1.3.0. 
https://github.com/lsst-ts/ts_hexapod_controller/blob/develop/doc/versionHistory.md

Thanks!

Reply:
Great suggestion. That's a good version number choice for such a significant change in ts_hexapod_controller.